api/api.go

```diff
@@ -1,23 +1,18 @@
 package api
 
 import (
-	"encoding/json"
-	"errors"
 	"fmt"
 	"net/http"
-	"strings"
 
 	"github.com/gorilla/mux"
 	"github.com/replicatedhq/embedded-cluster/api/controllers/auth"
 	"github.com/replicatedhq/embedded-cluster/api/controllers/console"
-	"github.com/replicatedhq/embedded-cluster/api/controllers/install"
+	linuxinstall "github.com/replicatedhq/embedded-cluster/api/controllers/linux/install"
 	"github.com/replicatedhq/embedded-cluster/api/docs"
+	"github.com/replicatedhq/embedded-cluster/api/handlers"
```

Comment:
Should `handlers` be a sub-package of `api/internal`? Is handlers something we want to expose outside of the EC module?